tck-dist/src/main/asciidoc/data-tck-reference-guide.adoc

```diff
@@ -20,7 +20,7 @@
 :JavaVersion1: 17
 :JavaVersion2: 21
 
-:license: Eclipse Foundation Technology Compatibility Kit License - v1.1
+:license: Eclipse Foundation Technology Compatibility Kit License
```

Comment:
Why is the v1.1 being removed here if we're switching to v1.1?

Reply:
(Unless the intent is to remove the versioning altogether, which would be fine. Seems like the rest of the repository uses EFTL and not EFTL v1.1)

Reply:
In the docs we provide a link to the license.  The link is static and will always point to the most recent license.